webapp/src/pages/applicationForm/components/ApplicationForm.js

```diff
@@ -207,6 +215,18 @@ class FieldEditor extends React.Component {
             errorText={validationError}
             required={question.is_required} />
         );
+      case MARK_DOWN:
+        return (
+          <div className="mark-down-form-wrapper">
+            <div id="md-editor" className={validationError && !answer ? "select-control is-invalid" : "md-editor select-control"}>
+              <MarkDownForm
+                onChange={(e) => this.handleChange(e)}
+                onImageUpload={(e) => this.handleUploadFile(e)}
+              />
+            </div>
+            <div className="invalid-feedback">An anwserer is required</div>
```

Comment:
This should be validationError, not a hard-coded message. Also, can you move the logic into component to keep it consistent with the other components?